lightbeam/__main__.py

```diff
@@ -140,18 +161,18 @@ def main(argv=None):
         )
     try:
         logger.info("starting...")
-        if args.command=='count': lb.counter.count()
-        elif args.command=='fetch': lb.fetcher.fetch()
-        elif args.command=='validate': lb.validator.validate()
-        elif args.command=='send': lb.sender.send()
-        elif args.command=='validate+send':
+        if args.command==ALL_COMMANDS['count']: lb.counter.count()
+        elif args.command==ALL_COMMANDS['fetch']: lb.fetcher.fetch()
+        elif args.command==ALL_COMMANDS['validate']: lb.validator.validate()
+        elif args.command==ALL_COMMANDS['send']: lb.sender.send()
+        elif args.command==ALL_COMMANDS['validate+send']:
             lb.validator.validate()
             lb.sender.send()
-        elif args.command=='delete': lb.deleter.delete()
-        elif args.command=='truncate': lb.truncator.truncate()
+        elif args.command==ALL_COMMANDS['delete']: lb.deleter.delete()
+        elif args.command==ALL_COMMANDS['truncate']: lb.truncator.truncate()
```

Comment:
... and these would have to be
```python
elif args.command=='truncate': lb.truncator.truncate()
```
maybe that's what you're trying to avoid.

I think I've seen some Python projects use constants in such situations, i.e.,
```python
COUNT_COMMAND='count'
FETCH_COMMAND='fetch'
...
ALL_COMMANDS = [ # maybe a better name is ALLOWED_COMMANDS?
  COUNT_COMMAND,
  FETCH_COMMAND,
  ...
]
...
if args.command==COUNT_COMMAND: lb.counter.count()
```

Reply:
Yeah I was torn between dict and list of constants. Maybe constants accomplishes equivalent future-proofing but it is more verbose and a little less airtight than a dict. I don't feel super strongly. Both are better than a list of strings IMO.

Reply:
That said, I do prefer `ALLOWED_COMMANDS` so I made that change